decentraland/components/presentational/Button.js

```diff
@@ -8,34 +8,56 @@ import {
   responsiveFontSize,
 } from "react-native-responsive-dimensions";
 
+// TODO: Change to NativeBase button component
+// https://github.com/tasitlabs/tasit/issues/204
 export default function Button(props) {
+  let { title, onPress, disabled } = props;
+  if (!disabled) disabled = false;
+  if (!onPress) onPress = () => {};
+  title = title.toUpperCase();
+
   return (
     <RNButton
-      containerStyle={styles.container}
-      style={styles.button}
-      onPress={props.onPress}
+      containerStyle={
+        disabled ? disabledStyles.container : enabledStyles.container
```

Comment:
I might have done `container.disabledStyle` personally so I could more quickly see what was different from the enabled version of the container. But this is a perfectly good choice too!

Reply:
Want to change it before merging or keep it the same?

Reply:
Done. It's much better now!

Reply:
Thanks! Resolving